frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -181,7 +181,6 @@ function BookInfo:show(file, book_props)
 
     local widget = KeyValuePage:new{
         title = _("Book information"),
-        value_overflow_align = "right",
```

Comment:
Why remove it from here ? (it was done in #3407).
Here, we would prefer `value_overflow_align = "right" `and `value_align = "left"`.